flutter_app/lib/widgets/display_all_logs_widget.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:imacs/modules/get_airside_logs.dart';
+import 'package:imacs/screens/log_displayer_screen.dart';
+
+class LogsList extends StatelessWidget {
+  const LogsList({
+    super.key,
+    required this.getAirsideLogs,
+  });
+
+  final GetAirsideLogs getAirsideLogs;
+
+  static Route _logDisplayerRoute(BuildContext context, Object? arguments) {
+    final String fileContent = arguments as String;
+    return MaterialPageRoute(
+      builder: (context) => LogDisplayerScreen(fileContext: fileContent),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      child: SizedBox(
+        width: 600,
+        child: AspectRatio(
+          aspectRatio: 4 / 3,
+          child: ListView.builder(
+            itemCount: getAirsideLogs.getFiles().length,
+            itemBuilder: (context, index) {
+              return Padding(
+                padding: const EdgeInsets.all(16.0),
+                child: ListTile(
+                  tileColor: Colors.blue.withOpacity(0.2),
```

Comment:
A easy way to style `ListTile` is to wrap them with the `Card` widget, this would respect the current Material3 theme